mdesnoyer/wonderland
# WU&S: {Desktop} Image Zoom #1272 + Make Lift component work #1322

# Changes
- better `gulpfle`
- retiring ‘ThumbBox`,`Thumbnail`,`Xylophone`and partial removal of
  `Modal`,`Hud`
- removing concept of enabled/disabled Thumbnails
- formatting and tidy up of reduced components
- killing some unused functions
- adding `pageOverlay` support
- killing dead translations
- `thumbnail_id` -> `thumbnailId`
- passing `togglePageOverlay` through to the right Thumbnails
- code tidy
- adding additional class helpers to thumbnails,
  `regular`/`neon`/`default` along with supporting CSS (aka `type`)
- fixing border back to transparent
- dumping `NA_STRING` and using `UNKNOWN_STRING` instead
- removing some `e`s
- added guard around `updateField`, known bug - #1320
- tweak to CSS to show/hide
- killed `Carousel` - doesn’t work with delivered HTML/CSS and no time
  to rework it all (TL;DR - navigation is inside the markup that we
  desire to show/hide)
- new component `ZoomThumbnail`
- moved paging into `Thumbnails`
- Supporting valence array
- wire up bars in Lift
- tweak to `makePercentage`
- added `portrait`, `landscape` and `square`
# Test Plan
- process some videos, check out Image Zoom
- make sure Lift is consistent between Results and Image Zoom
- check for dragons
